app/views/api/v1/labels/most_used.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.data do
```

Comment:
Please don't use this. On frontend this would become `response.data.data.payload`